### PR TITLE
Feat: Update with expanded LineOrg API

### PR DESCRIPTION
**Pull request by larfeq**
- [ ] New feature
- [ ] Bug fix
- [ ] High impact
- [x] Improvement

**Work description**:

We found out that we could expand LineOrg call to include the recipients so we can drop the extra call resolving notification recipients.

**Expected result**: A more effective function

Link to workitem: [LINK](https://statoil-proview.visualstudio.com/Fusion%20Resource%20Allocation/_workitems/edit/54652)

**Test description**:

- [ ] Can be tested
- [ ] Automatic tests created / updated
- [x] Local tests are passing

**Checklist**:

- [ ] Considered automated tests
- [ ] Considered updating specification / documentation
- [ ] Considered work items
- [ ] Considered security
- [x] Performed developer testing
- [x] Checklist finalized / ready for review

### CODE REVIEW

File: src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs [hunk 1]
```diff
@@ -49,7 +50,7 @@ public class ScheduledReportTimerTriggerFunction
 
     [FunctionName("scheduled-report-timer-trigger-function")]
     public async Task RunAsync(
-        [TimerTrigger("0 10 0 * * MON", RunOnStartup = false)]
+        [TimerTrigger("0 10 0 * * MON", RunOnStartup = true)]
```
[the remark this answers is not in the document]
Comment (larfeq) [reply src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs]: Fixd

**Review by aleklundeq — APPROVED** [review]
Looks good, but we should wait with merging and deploying this to CI until after summer vacations

File: src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs [hunk 2]
```diff
@@ -83,41 +88,32 @@ private async Task SendResourceOwnersToQueue(ServiceBusSender sender, int totalB
...
-
-                    // Create a new OrgUnits object with the FullDepartment, SapId, and combined Person array
-                    return new OrgUnits
-                    {
-                        FullDepartment = group.Key,
-                        SapId = group.First().SapId,
-                        Management = new Management { Persons = allPersons }
-                    };
-                })
+                .Where(d => d.FullDepartment.Contains("PDP PRD PMC PCA PCA5"))
```
Comment (aleklundeq) [inline src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs:91]: For testing puposes? Should remove this before merging

File: src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (aleklundeq) [reply src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs:58]: Is this resolved

File: src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs (hunk 2 above)
Comment (larfeq) [reply src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs]: Fixed

File: src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (larfeq) [reply src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ResourceOwner/WeeklyReport/ScheduledReportTimerTriggerFunction.cs]: Yes! :)

**Review by aleklundeq — APPROVED** [review]
LGTM

**Pull request merged**